setup/include/inc.setup_header.php

```diff
@@ -55,11 +55,6 @@
 require_once "./include/inc.ilias_version.php";
 include_once './Services/Logging/classes/public/class.ilLogLevel.php';
 
-// include error_handling
-require_once "./setup/classes/class.ilSetupErrorHandling.php";
-
-$ilErr = new ilSetupErrorHandling();
-$ilErr->setErrorHandling(PEAR_ERROR_CALLBACK,array($ilErr,'errorHandler'));
```

Comment:
This ain't it. `$ilErr` is used later on (l. 144 in the current trunk). Also: where is `$GLOBALS["ilErr"]` initialized?